pmlc/rt/level_zero/level_zero_device.cc

```diff
@@ -0,0 +1,106 @@
+// Copyright 2020 Intel Corporation
+#include "pmlc/rt/level_zero/level_zero_device.h"
+
+#include "pmlc/rt/jit_executable.h"
+#include "pmlc/util/logging.h"
+
+namespace pmlc::rt::level_zero {
+
+LevelZeroQueue::LevelZeroQueue(const ze_context_handle_t &context,
+                               const ze_device_handle_t &device,
+                               ze_command_queue_group_properties_t properties)
+    : properties(properties) {
+  queue = lzu::create_command_queue(context, device, 0,
```

Comment:
You might want to add llvm style comments `/*description*/` for these 0

Reply:
Done